### ISSUE TITLE
Problems with undefined or null types

### ISSUE THREAD

**Issue by nidomiro** [epeli/babel-plugin-ts-optchain#1]
I'm trying to use this awsome lib (macro) with create-react-app, but have some difficulties:

## Function value: undefined
```
import { oc } from "ts-optchain.macro";

interface A {
    b?: string | null
}

function fun(a: A | null | undefined){
    oc(a).b();
}

fun(undefined);
```

### Result:
```
Failed to compile.

./src/index.tsx

TypeError: ts-optchain.macro: Cannot read property 'path' of undefined Learn more: https://www.npmjs.com/package/ts-optchain.macro

```

## Function value: null

```
import { oc } from "ts-optchain.macro";

interface A {
    b?: string | null
}

function fun(a: A | null | undefined){
    oc(a).b();
}

fun(null);
```

### Result:
```
Failed to compile.

./src/index.tsx
TypeError: ts-optchain.macro: Cannot read property 'path' of undefined Learn more: https://www.npmjs.com/package/ts-optchain.macro

```

## Function value: Object
```
import { oc } from "ts-optchain.macro";

interface A {
    b?: string | null
}

function fun(a: A | null | undefined){
    oc(a).b();
}

fun({});
```

### Result:
```
Failed to compile.

./src/index.tsx
TypeError: ts-optchain.macro: Cannot read property 'path' of undefined Learn more: https://www.npmjs.com/package/ts-optchain.macro

```

## Function value: working
```
oc({}).toString();
```

### Result:
```
works
```

**Comment by esamattis** [comment]
Thanks for reporting! Should be fixed in 1.1.3

**Comment by nidomiro** [comment]
Thank you very much :) Its working now - even in ie11 ;)
